src/main/java/cr0s/warpdrive/AndroidCheck.java

```diff
@@ -0,0 +1,10 @@
+package cr0s.warpdrive;
```

Comment:
This would make more sense as a CompatMatterOverdrive class with the other compatibility classes.

Reply:
Well, all the classes in the `compat` package are implementing `IBlockTransformer`, so I decided that putting a completely random class here won't make any sense. As you want, I'll change this then.